data-products/_docs/product-governance.md

```diff
@@ -0,0 +1,109 @@
+# Defining data product governance
+
+The purpose of the `01-governance.yml` file in your data product definition folder is to indicate to users, and to the data platform itself, how the data should be handled, and where it has come from. A correctly specficied set of governance attributes gives you and platform users confidence in handling and using the data.
+
+## Example
+
+```yaml
+governance:
+  data-platform.justice.gov.uk/data-controller: "moj"
+  data-platform.justice.gov.uk/review-after: "2023-12-31"
+  data-platform.justice.gov.uk/siro: "information.asset.owner.name@justice.gov.uk"
+  data-platform.justice.gov.uk/protective-marking: "None"
+  data-platform.justice.gov.uk/status: "production"
+  data-platform.justice.gov.uk/allow-derived-products: true
```

Comment:
Maybe remove this option. We can add it later on when/if we need it

Reply:
Mind if I leave it? A fair amount of this is aspirational, and/or markers for things we'd _like_ to do.  Like a lot of the metadata, it's about making users at least _think_ about this stuff.

Reply:
I don't mind it :-)
I do question what the purpose of it is though. Is there any reasoning on not allowing a derived product? This sounds counter intuitive with the purpose of the mesh, and not something that I believe should be made easy for people to do.
What does it mean if I say that I don't allow derived products? Is it really enforceable?

Reply:
Apologies, thought you meant all 4 lines. Fair enough, maybe I got a case of the YAGNIs - I'll remove it